eng/pipelines/templates/steps/code-coverage.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  ServiceDirectory: ''
+  Enabled: ''
+
+steps:
+  # only run when Enabled has a value. Coverage can be disabled by service by setting service_disable_coverage env var
+  - ${{ if ne(parameters.Enabled, '') }}:
```

Comment:
I think you might want to change this to a runtime condition as you cannot access runtime variables during template expansion. While this template itself is fine you are trying to condition it based on a matrix variable above and that will not work.